### DISCUSSION TITLE
essay/2024/04/15/change-writing

### DISCUSSION THREAD

**Discussion by giscus** [BenjaminKim/benjaminkim.github.io#205] [Comments]
# essay/2024/04/15/change-writing

제 블로그가 완전히 변하게 된 계기가 세 번 있었습니다.

https://jeho.page/essay/2024/04/15/change-writing.html

**Comment by Invidam** [comment]
글쓰기 방식 정말 매력있으십니다 ㅋㅋㅋ 

저는 최근에 스티븐 킹의 `유혹하는 글쓰기`라는 책을 봤는데 단순함을 무척 강조하더라구요. 뛰어난 작가들은 통하는 게 있나봅니다.

**Reply by BenjaminKim** [reply]
칭찬 고맙습니다...! 🥰
